kombu/transport/zookeeper.py

```diff
@@ -141,7 +140,7 @@ def _has_queue(self, queue):
 
     def _open(self):
         conninfo = self.connection.client
-        self.vhost = os.path.join('/', conninfo.virtual_host[0:-1])
```

Comment:
In ZooKeeper domain `virtual_host` (aka `chroot`) must be a string. It will be a prefix of a path to set a value for (Ctrl+F at [this doc page](https://zookeeper.apache.org/doc/r3.2.2/zookeeperProgrammers.html)).

I suppose `[0:-1]` slicing had the aim to prevent from chroots with trailing slashes, e.g. `/chroot/`.